test/Reverts.ts

```diff
@@ -23,13 +28,22 @@ describe("Reverts test", function () {
     jommys = await ethers.getImpersonatedSigner(
       "0x0AeaC6D1424EA6d0F87123A50CA5eEc9f16108c5",
     );
-    MRC = await ethers.getContractAt(
-      "IMRC",
-      "0xeF453154766505FEB9dBF0a58E6990fd6eB66969",
-    );
+    if (env.TEST_LOCAL_BLOCKCHAIN) {
+      const { e7lContract, mrcContract } = await deployContracts();
+      E7L = e7lContract;
+      MRC = mrcContract;
+
+      await MRC.mint(1);
+      await MRC.connect(yonathan).mint(1);
+    } else {
+      MRC = await ethers.getContractAt(
+        "MRCRYPTO",
+        "0xeF453154766505FEB9dBF0a58E6990fd6eB66969",
```

Comment:
This is an opinion. Normally when you have `magic numbers` or magic variables, they are usually extracted to private constants to give context.

Reply:
Well seen